SPECS/gmp/gmp.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Math libraries
 Name:           gmp
 Version:        6.2.1
-Release:        2%{?dist}
+Release:        3%{?dist}
```

Comment:
This change impacts the toolkit.
I think we should also bump the appropriate toolkit manifest entries as well & mark `Yes` in the checklist `Does this affect the toolchain?`

Consider bumping entries in the following 4 files:
```
toolkit/resources/manifests/package/pkggen_core_{x86_64,aarch64}.txt

toolkit/resources/manifests/package/toolchain_{x86_64,aarch64}.txt
```